ClosedXML/Excel/Tables/XLTable.cs

```diff
@@ -817,5 +817,39 @@ public DataTable AsNativeDataTable()
 
             return table;
         }
+
+        public IXLTable CopyTo(IXLWorksheet targetSheet)
+        {
+            return CopyTo((XLWorksheet) targetSheet);
+        }
+
+        internal IXLTable CopyTo(XLWorksheet targetSheet)
+        {
+            String tableName = Name;
+            var newTable = (XLTable)targetSheet.Table(targetSheet.Range(RangeAddress.ToString()), tableName, true);
+
+            newTable.RelId = RelId;
```

Comment:
The `RelId` is to store the `relId` when a workbook is opened. When creating a new table (by copying it), the `RelId` should remain empty.

Reply:
Perhaps you are right. I just moved this from `XLWorksheet` class. And this code was quite ancient (2011, commit message "worksheet.CopyTo(...) should now work as expected." LOL)

Reply:
If it's from 2011, then it's before my time. Let me just double-check my previous statement in this regard first.

Reply:
I'll add another test tomorrow.

Reply:
When I remove this line `CopyWorksheets` test fails. The produced file is correct either way, just table IDs differ.